README.md

```diff
@@ -113,6 +113,12 @@ const support = bcd.css.properties.background.__compat;
 const support = bcd['api']['Document']['body']['__compat'];
 ```
 
+### TypeScript Support
```

Comment:
Why is this in the "usage" section? My initial read had me thinking this slotted in better the import stuff and "other languages" but maybe I'm missing something here.